experimental/dds/tree2/src/test/domains/json/twitter.ts

```diff
@@ -22,30 +22,73 @@ import {
  * json benchmarks twitter.json - https://raw.githubusercontent.com/serde-rs/json-benchmark/master/data/twitter.json
  */
 // Shared tree keys that map to the type used by the Twitter type/dataset
-export const Twitter = {
-	SharedTreeFieldKey: {
-		statuses: brand<FieldKey>("statuses"),
-		retweetCount: brand<FieldKey>("retweet_count"),
-		favoriteCount: brand<FieldKey>("favorite_count"),
-	},
+export const TwitterKey = {
+	statuses: brand<FieldKey>("statuses"),
+	retweetCount: brand<FieldKey>("retweet_count"),
+	favoriteCount: brand<FieldKey>("favorite_count"),
 };
```

Comment:
Originally, 'Twitter' was both a type and an object exposing the field keys.

I probably could have left this as is, but thought it was better to separate them to avoid confusion if anyone uses this as an example.